web/ui/react-app/package.json

```diff
@@ -37,20 +36,18 @@
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.7.0",
     "react-resize-detector": "^5.0.7",
-    "react-scripts": "3.4.4",
     "react-test-renderer": "^16.9.0",
     "reactstrap": "^8.9.0",
     "sanitize-html": "^2.3.3",
     "sass": "1.32.10",
     "tempusdominus-bootstrap-4": "^5.1.2",
     "tempusdominus-core": "^5.0.3",
-    "typescript": "^3.3.3",
     "use-media": "^1.4.0"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --runInBand",
+    "test": "react-scripts test --runInBand --resetMocks=false",
```

Comment:
this weird thing is coming from this issue : https://github.com/facebook/create-react-app/issues/10571#issuecomment-784923197